contrib/create-coreos-vdi

```diff
@@ -39,6 +39,13 @@ if [ $? -ne 0 ]; then
     exit 1
 fi
 
+# GPG tools required
+which gpg &>/dev/null
+if [ $? -ne 0 ]; then
```

Comment:
In general, you'd want to wrap vars with double quotes `"$?"`. It's just that it this case it would always have some return code, but for arbitrary var it might end up being interpolated as `if [  -ne 0 ]; then` in case of empty var/string, which would lead to syntax error being raised by shell interpreter.